src/gui_boxmenu.c

```diff
@@ -63,7 +63,9 @@ long gf_all_researchable(struct GuiBox *gbox, struct GuiBoxOption *goptn, unsign
 long gfa_can_give_controlled_creature_spells(struct GuiBox *gbox, struct GuiBoxOption *goptn, long *tag);
 long gfa_controlled_creature_has_instance(struct GuiBox *gbox, struct GuiBoxOption *goptn, long *tag);
 long gf_decide_victory(struct GuiBox *gbox, struct GuiBoxOption *goptn, unsigned char btn, long *tag);
-TbBool gfa_single_player_mode(struct GuiBox *gbox, struct GuiBoxOption *goptn, unsigned char btn, long *tag);
+long gfa_single_player_mode(struct GuiBox *gbox, struct GuiBoxOption *goptn, unsigned char btn, long *tag);
```

Comment:
I'm getting a build warning. Perhaps caused by this line?

![image](https://user-images.githubusercontent.com/13840686/163886167-a6145d38-4aab-48f4-a3b9-973e4eb716ec.png)

Reply:
I don't think it's that line itself, but the fact it returns a long. No idea what to do about it though.....